plugins/apache_druid_logs.yaml

```diff
@@ -0,0 +1,296 @@
+version: 0.0.1
+title: Apache Druid
+description: Log parser for Apache Druid
+parameters:
+  - name: enable_broker_logs
+    description: Enable to collect Apache Druid broker logs
+    type: bool
+    default: true
+  - name: broker_log_path
+    description: Absolute filepath containing Apache druid broker logs. Required if enable_broker_logs is true
+    type: "string"
+  - name: enable_coordinator_overlord_logs
+    description: Enable to collect Apache Druid coordinator and overlord logs
+    type: bool
+    default: true
+  - name: coordinator_overlord_log_path
+    description: Absolute filepath containing Apache Druid coordinator and overlord logs. Required if enable_coordinator_overlord_logs is true
+    type: "string"
+  - name: enable_historical_logs
+    description: Enable to collect Apache Druid historical logs
+    type: bool
+    default: true
+  - name: historical_log_path
+    description: Absolute filepath containing Apache Druid historical logs. Required if enable_historical_logs is true
+    type: "string"
+  - name: enable_middle_manager_logs
+    description: Enable to collect Apache Druid middle manager logs
+    type: bool
+    default: true
+  - name: middle_manager_log_path
+    description: Absolute filepath containing Apache Druid middle manager logs. Required if enable_middle_manager_logs is true
+    type: "string"
+  - name: enable_router_logs
+    description: Enable to collect Apache Druid router logs
+    type: bool
+    default: true
+  - name: router_log_path
+    description: Absolute filepath containing Apache Druid router logs. Required if enable_router_logs is true
+    type: "string"
+  - name: enable_zookeeper_logs
+    description: Enable to collect Apache Druid ZooKeeper logs
+    type: bool
+    default: true
+  - name: zookeeper_log_path
+    description: Absolute filepath containing Apache Druid ZooKeeper logs. Required if enable_zookeeper_logs is true
+    type: "string"
+  - name: start_at
+    description: At startup, where to start reading logs from the file (`beginning` or `end`)
+    type: string
+    valid_values:
+     - beginning
+     - end
+    default: end
+  # Apache Druid logs do not include a timezone in their timestamps, so we need to set it explicitly.
+  - name: timezone
+    description: Timezone to use when parsing the timestamp
```

Comment:
Description should state the condition that the logs timestamp don't contain a timezone.